wp-content/themes/core/components/navigation/js/site-header.js

```diff
@@ -0,0 +1,278 @@
+/**
```

Comment:
Nice work cleaning this up and separating Nav and Header/Masthead functionality.